sdk/instrumentation/instrumentation-opentelemetry/README.md

```diff
@@ -0,0 +1,94 @@
+# Azure OpenTelemetry Instrumentation library for JavaScript
+
+## Getting started
+
+### Currently supported environments
+
+- [LTS versions of Node.js](https://nodejs.org/about/releases/)
+- Latest versions of Safari, Chrome, Edge, and Firefox.
+
+See our [support policy](https://github.com/Azure/azure-sdk-for-js/blob/main/SUPPORT.md) for more details.
+
+### Prerequisites
+
+- An [Azure subscription][azure_sub].
+- The [@opentelemetry/instrumentation][otel_instrumentation] package.
+
+You'll need to configure the OpenTelemetry SDK in order to produce Telemetry data. While configuring OpenTelemetry is outside the scope of this README, we encourage you to review the [OpenTelemetry documentation][otel_documentation] in order to get started using OpenTelemetry.
+
+### Install the `@azure/instrumentation-opentelemetry` package
+
+Install the Azure OpenTelemetry Instrumentation client library with `npm`:
+
+```bash
+npm install @azure/instrumentation-opentelemetry
+```
+
+### Browser support
+
+#### JavaScript Bundle
+
+To use this client library in the browser, first you need to use a bundler. For details on how to do this, please refer to our [bundling documentation](https://aka.ms/AzureSDKBundling).
+
+## Key concepts
+
+- The **createAzureInstrumentation** function is the main hook exported by this library which provides a way to create an Azure SDK Instrumentation object which can be registered with OpenTelemetry.
+
+### Compatibility with existing Client Libraries
+
+- TODO, we should describe what versions of core-tracing are compatible here...
+
+## Examples
+
+### Enable OpenTelemetry instrumentation
+
+```javascript
+const { registerInstrumentations } = require("@opentelemetry/instrumentation");
+const { createAzureInstrumentation } = require("@azure/instrumentation-opentelemetry");
```

Comment:
Consider naming here - do we conform to our naming or stay consistent with OTel's naming? Can be confusing with the rest of the OTel ecosystem.

When I register it what exactly is getting instrumented? How is this being used? What are the effects of calling this?